tests/installation/welcome.pm

```diff
@@ -19,6 +19,28 @@ use utils 'ensure_fullscreen';
 use version_utils qw(is_sle sle_version_at_least);
 use main_common 'is_staging';
 
+sub check_keyboard_layout {
+    if (get_var('INSTALL_KEYBOARD_LAYOUT')) {
```

Comment:
Instead of wrapping the whole block in an if how about returning early with unless?:

```perl
return unless get_var('INSTALL_KEYBOARD_LAYOUT');
…
```